mitodl/open-discussions
# Fixed scrollbar-on-hover for the drawer

#### Pre-Flight checklist

#### What are the relevant tickets?
No ticket

#### What's this PR do?
Fixes bug introduced by #1247

#### How should this be manually tested?
Use the drawer at different browser widths and, with enough channels that a scrollbar is required, click on channel links that appear low on the list

#### Any background context you want to provide?
- The separation of drawer styles between `drawer.scss` and `navigation.scss` was causing some problems, and it didn't make much sense to me to begin with. I started editing `navigation.scss` to prevent a situation where the two files were defining styles for the same class that only appeared in the drawer, then ended up moving all drawer-specific styles from `navigation.scss` to `drawer.scss`
- I will close the revert PR if this one merges (#1270)

#### Screenshots (if appropriate)
![ss 2018-09-27 at 12 51 36](https://user-images.githubusercontent.com/14932219/46162193-93a6f000-c255-11e8-94db-e0adb74d925e.png)
![ss 2018-09-27 at 12 51 20](https://user-images.githubusercontent.com/14932219/46162194-93a6f000-c255-11e8-8a84-d94ea0094d34.png)
![ss 2018-09-27 at 13 02 31](https://user-images.githubusercontent.com/14932219/46162212-9dc8ee80-c255-11e8-9307-b450ebde1657.png)
